### PR TITLE
Initial effort at Group, ClusterRole and ClusterRoleBinding for osd-devaccess

### PR DESCRIPTION
**Pull request by robotmaxtron**
As part of OSD-3282/3284

### CODE REVIEW

File: deploy/osd-devaccess/01-osd-devaccess.ClusterRole.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  # devaccess can get, list, and watch all resources
+  - apiGroups:
+      - "*"
+    resources:
+      - "*"
+    verbs:
+      - get
+      - list
+      - watch
+  # devaccess can get and list nodes
```
Comment (cblecker) [inline deploy/osd-devaccess/01-osd-devaccess.ClusterRole.yaml:15]: Are all these get/list entries for other objects redundant if the first rule is "get/list/watch all"?

Comment (robotmaxtron) [reply deploy/osd-devaccess/01-osd-devaccess.ClusterRole.yaml:15]: Yep, thank you.

**Pull request closed without merging**